web_widget_text_markdown/README.rst

```diff
@@ -11,7 +11,7 @@ This module adds a new widget for text field in form view on Odoo:
 Installation
 ============
 
-It was tested on openerp trunk, 8.0 branch.
+It was tested on openerp trunk, 10.0 branch.
```

Comment:
Remove this section, it makes not much sense, the addon is set to instalable, so it's implicitly expected to work